tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/visibility/retry/RetryServiceMethodSuppressLevel.java

```diff
@@ -0,0 +1,37 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2016-2017 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.eclipse.microprofile.fault.tolerance.tck.visibility.retry;
+
+import java.io.IOException;
+import java.sql.Connection;
+
+import javax.enterprise.context.RequestScoped;
+
+@RequestScoped
+@RS(RetryServiceType.METHOD_SUPPRESS)
+public class RetryServiceMethodSuppressLevel extends BaseRetryService {
```

Comment:
Don't forget that since the FT annotations are all `@Inherited` the original annotations will be retained.  E.g. calling `RetryServiceMethodSuppressLevel.class.getMethod("service").getAnnotation(Retry.class)` will still return the object.

I don't think your test for this will pass for anyone.

Reply:
@johnament this came from #183 and the answer of from @Emily-Jiang to your issue.

I have no pb in keeping it or removing it

> I don't think your test for this will pass for anyone.

unless implementations use `getDeclaredAnnotation()` instead of `getAnnotation()`

Honestly, I see interests in both way of handling this annotation retrieval.
It is a matter of decision & balance between the strict conformity & the facility offer to disable a feature by overriding a method.
Moreover, looking at the javadoc of `@Inherited`, no strict description is done when it comes to method annotation lookup.

If there were votes available on an issue in github we could go with a democratic way ; instead I'd expect project leads to take a decision.

Reply:
Well, the project leads would be John C or Kevin.  I'm not in agreement with Emily's POV, since these are `@Inherited`.  I would recommend that we remove `@Inherited` if this is the behavior expected.

Reply:
Since the subclass inherits the annotations, the method will have the annotations in RetryServiceMethodSuppressLevel. Actually, think it further. It might be better not to make them inherited, because we cannot easily get rid of the annotations. If they are not inherited, we can always add them. We can discuss this on tomorrow's (15th Nov) call as well.

Reply:
Isn't that what I had just recommended, removing `@Inherited`?

Reply:
Yes, John. I think I am not in disagreement with you on this. Since this is behaviour changes, we should have further discussion on tomorrow's call.